### PR TITLE
starting mobile version from 0

### PR DESCRIPTION
**Pull request by latinogan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="index.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+	<title>setup mobile version skeleton</title>
+</head>
+ 
+
+<body>
```
Comment (Whoistolu) [inline index.html:13]: - [ ] [REQUIRED] Please check the width of the body as you have horizontal scroll bars. kindly ensure your design looks more like the provided design in the template you have chosen.
Screenshot from 2022-02-16 21-52-45
Screenshot from 2022-02-16 21-52-51

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @latinogan,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: csssss [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+.topnav #myLinks {
+  display: none;
+}
+
+a {
+  float: left;
+  color: white;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
```
Comment (medaminedev66) [inline csssss:18]: - [ ] Kindly remove the unused `csssss` file so your repository is clean.

File: index.html [hunk 3]
```diff
@@ -10,15 +10,15 @@
 </head>
  
 
-<body>
+<e>
     <header>
```
Comment (medaminedev66) [inline index.html]: - [ ] Kindly remove the gap from the top and the left of your page to improve your UI design.
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided. I would recommend using the same fonts that are provided in the Figma design.
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] Kindly add a pull request description of what you have done in this pull request so that others can read what is being worked on.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: csssss (hunk 2 above)
Comment (latinogan) [reply csssss:18]: ohh sorry about that i though i deleted before send the pull request

**Review by JuliCarracedo — APPROVED** [review]
Hi @latinogan , 
    
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
    Congratulations! 🎉 

### Acknowledgement 
You did a great job so far! I'm aware you are just starting Microverse, so I won't demoralize you by rejecting your project. But there's still a ton of room for improvement. Portfolios are an important part of the professional environment since they show not only your skill but also your philosophy and work ethic.
I've left some comments that will help you improve your app. You are completely free to either follow them or not.
    
Cheers and Happy coding!👏👏👏
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
    Please, remember to tag me in your question so I can receive the notification.
    
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**